package.json

```diff
@@ -64,6 +64,7 @@
     "filetype": "./content/src/filetype.js",
     "gadget": "./content/src/gadget.js",
     "guide": "./content/src/guide.js",
+    "html2canvas": "./content/lib/html2canvas.js",
```

Comment:
I think we discussed this before, but I don't remember.  Is it possible to bring in html2canvas via npm?